atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/ValueType.java

```diff
@@ -889,7 +890,12 @@ public Class<?> getJavaObjectClass() {
     }
 
     public String getJavaClassName() {
-        return getJavaClass().getSimpleName();
+        Class<?> clazz = getJavaClass();
+        String simpleName = clazz.getSimpleName();
+        if (clazz.isPrimitive()) {
+            return simpleName.toLowerCase(Locale.ROOT);
+        }
+        return simpleName;
     }
```

Comment:
The main concern here is whether the `ValueType` goes directly into a Collection type thing elsewhere. I'm pretty sure it doesn't (we wrap things into `MyTableRow/Column/ColumnValue`) so this should be ok